### PR TITLE
Add new_severity to override mutations

### PR DESCRIPTION
**Pull request by swaterkamp**
**What**:
Add missing field new_severity to create and modify mutations.

**Why**:
new_severity was missing in the original schema

special cases are sent to gvmd to, e.g. select "any task". This fails if we only expect true UUIDs

**How**:

**Checklist**:

- [x] Tests
- [x] [CHANGELOG](https://github.com/greenbone/hyperion/blob/master/CHANGELOG.md) Entry
- [x] Documentation

### CODE REVIEW

**Review by bjoernricks — CHANGES_REQUESTED** [review]
Changing result_id and task_id from UUID to string should not be done. The type of a field should be as concrete as possible. You spotted the case where we need to reset a field. We need to define how to handle this.

And also both cases aren't tested yet. Such changes must be tested.

File: selene/schema/overrides/mutations.py [hunk 1]
```diff
@@ -156,12 +168,15 @@ class ModifyOverrideInput(graphene.InputObjectType):
     hosts = graphene.List(
         graphene.String, description="A list of hosts addresses"
     )
+    new_severity = graphene.Float(
+        required=True, description="Severity to which should be overridden"
+    )
```
Comment (bjoernricks) [inline selene/schema/overrides/mutations.py:173]:
```suggestion
    new_severity = graphene.Field(
        SeverityType,
        required=True, description="Severity to which should be overridden"
    )
```

**Review by bjoernricks — CHANGES_REQUESTED** [review]

File: selene/schema/overrides/mutations.py (hunk 1 above)
Comment (swaterkamp) [reply selene/schema/overrides/mutations.py]: Implemented with additions in https://github.com/greenbone/hyperion/pull/85/commits/e6f48dc60115afd3a13598aead75bab52a978516

File: selene/schema/overrides/mutations.py [hunk 2]
```diff
@@ -61,7 +63,14 @@ class CreateOverrideInput(graphene.InputObjectType):
...
         text (str): Text of the new override
-        nvt (NVT): The NVT of the override
+        nvt_oid (str): The NVT of the override
+        days_active (int): Days override will be active. -1 on always, 0 off
+        hosts (str): A list of host addresses
+        new_severity (float): Severity to which should be overridden
+        port (str): Port to which the override applies
+        result_id (UUID): ID of a result to which override applies
+        severity (float): Severity to which override applies
+        task_id (UUID): ID of a task to which override applies
```
Comment (bjoernricks) [inline selene/schema/overrides/mutations.py:73]: No need to add these docstrings in future anymore. The argument docs will be created from the description of the fields.

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**